registry/registry.go

```diff
@@ -168,6 +168,7 @@ const (
 var (
 	typeStrings       = [...]string{"int", "number", "string", "bool", "map", "array", "any", "alt"}
 	ValidRegistryName = regexp.MustCompile("^[a-z0-9](?:-?[a-z0-9])*$")
+	ValidViewName     = regexp.MustCompile(fmt.Sprintf("^%s$", subkeyRegex))
```

Comment:
`subkeyRegex` is defined in a `var ( ... )` block around line 375 along with some other validation regexes, perhaps those definitions should be moved up here so they're all in one place and it's clearer how `ValidViewName` compares to `ValidRegistryName`.

Reply:
Sure, those use only be used to validate path elements so they were closer to those functions. I've now moved them to the same block. Thanks